<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Perceptrón</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTazaAprendizaje">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>η (Taza de aprendizaje)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="n1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>Ingrese η1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="n2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>Ingrese η2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="n3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>Ingrese η3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="n4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>Ingrese η4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="n5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>Ingrese η5</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblValoresW">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Valores de W</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="w1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>Ingrese Valor de W1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="w2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>Ingrese Valor de W2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="w3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>Ingrese Valor de W3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="aleatorioT">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Generar Todo Aleatoriamente</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="definirValores">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Agregar Valores</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIniciarProceso">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>241</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Iniciar Proceso De Entrenamineto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sallir</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbOpcion">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>741</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Consolas&quot;;
font: 28pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
